deploy/chart/devfile-registry/README.md

```diff
@@ -21,17 +21,84 @@ E.g. if your cluster's ingress domain is 192.168.1.0.nip.io, you would run:
 helm install devfile-registry deploy/chart/devfile-registry --set global.ingress.domain=192.168.1.0.nip.io
 ```
 
+### Kubernetes Installation Examples
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \ 
+    --set global.ingress.domain=<ingress-domain> \
+	--set devfileIndex.image=<index-image> \
+	--set devfileIndex.tag=<index-image-tag>
+```
+
+Where `<ingress-domain>` is the ingress domain for your cluster, `<index-image>` is the devfile index image you want to deploy, and `<index-image-tag>` is the corresponding image tag for the devfile index image.
+
+For example, if you're installing your own custom devfile registry image for dev/test purposes on Minikube, you might run:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest
+```
+
+You can deploy a devfile registry with a custom registry viewer image (uses `quay.io/devfile/registry-viewer:next` by default) by running the following:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest \
+	--set registryViewer.image=quay.io/someuser/registry-viewer \
+	--set registryViewer.tag=latest
+```
+
+You can deploy a *headless* devfile registry (i.e. without the registry viewer) by specifying `--set global.headless=true` which will look like:
+
+```bash
+$ helm install devfile-registry ./deploy/chart/devfile-registry \
+    --set global.ingress.domain="$(minikube ip).nip.io" \
+	--set global.headless=true \
+	--set devfileIndex.image=quay.io/someuser/devfile-index \
+	--set devfileIndex.tag=latest
+```
+
 ## Installing the Devfile Registry on OpenShift
 
-If you're installing on OpenShift, you need to set `global.isOpenShift` to true, for example:
+If you're installing on OpenShift, you will first need to set `global.isOpenShift` to true, for example:
```

Comment:
Nit: We're setting `isOpenShift` to true during install, so I don't think it's been set _first_?